.pipelines/cni/cilium/cilium-scale-test.yaml

```diff
@@ -0,0 +1,228 @@
+pr: none
+trigger: none
+
+stages:
+  - stage: update_cilium_version
+    displayName: "Update Cilium Version and Restart Nodes"
+    jobs:
+      - job: update_version
+        pool:
+          name: "$(BUILD_POOL_ACN)"
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(TEST_SUB_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                az aks get-credentials --resource-group ${CLUSTER} --name ${CLUSTER}
```

Comment:
We only have one cluster that we use for the scale tests, and as of now we expect to use the same one each time. Multiple builds is not a concern for now. I set the cluster name as default for the var so we don't have to change it at each run, but it is mutable in case we ever do need to use a different cluster.